### PR TITLE
Get the namespace from the git remote url

### PR DESCRIPTION
**Pull request by singularo**
Make the build.sh script nicer.

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-NAMESPACE="uofa"
+NAMESPACE=$(git remote get-url origin | awk -F'[/:]' '{ print $2 }')
```
Comment (pingers) [inline build.sh:2]: This just has to pass the "Does it work on a Mac?" test.

Comment (bart88) [reply build.sh:2]: `-F` flag option is available on mac. Running 
`git remote get-url origin | awk -F'[/:]' '{ print $2 }'` on this repo produces : `universityofadelaide` 

🎉  👍

**Review by bart88 — APPROVED** [review]
🚢  Thanks for the Contribution @singularo

File: build.sh (hunk 1 above)
Comment (singularo) [reply build.sh:2]: Yeah, now I know about multiple field delimeters in awk, I know a few places that will come in handy!

**Pull request merged**